engine/main.py

```diff
@@ -528,7 +528,11 @@ def add_members(group_name, emails, add_as_mods, user):
             membergroup = MemberGroup.objects.get(group=group, member=user)
             is_admin = membergroup.admin
         if is_public or is_admin:
-            email_list = emails.strip().lower().split(',')
+            emails_txt = emails.strip().lower()
+            email_serators = [emails_txt.split(','), emails_txt.split(';'), emails_txt.split('|'), emails_txt.split('/')]
+            email_serators_len = [len(x) for x in email_serators]
+            seperator_index = email_serators_len.index(max(email_serators_len))
+            email_list = email_serators[seperator_index]
```

Comment:
this doesn't work if users mix the possible separators in the same string.

Reply:
Yes. That's intentional. People don't mix up the separators within one list of email addresses

Reply:
Done!